packages/components/input/src/use-input.ts

```diff
@@ -516,7 +520,13 @@ export function useInput<T extends HTMLInputElement | HTMLTextAreaElement = HTML
         "aria-label": "clear input",
         "data-slot": "clear-button",
         "data-focus-visible": dataAttr(isClearButtonFocusVisible),
-        className: slots.clearButton({class: clsx(classNames?.clearButton, props?.className)}),
+        className: slots.clearButton({
+          class: clsx(
+            classNames?.clearButton,
+            props?.className,
+            isMultiline ? "relative block opacity-100 p-0 -m-0 end-0" : "",
```

Comment:
why we need these classes? feel like none is required.

Reply:
Make the button relatively positioned and have its position placed in the correct place

Reply:
if i delete `relative block opacity-100 p-0 -m-0 end-0`, the clear button is still in the same place.

Reply:
That's because I've already used these styles to override the old ones, so the position won't change if I remove these attributes

Reply:
![image](https://github.com/user-attachments/assets/fc253c3e-c5f5-4e92-b976-30e0260f62a6)
These are the default styles